README.md

```diff
@@ -46,7 +46,10 @@ import { createClient } from 'redis';
 
 const client = createClient();
 
-client.on('error', (err) => console.log('Redis Client Error', err));
+client.on('error', (err) => {
+    // Do **NOT** throw here as it would break auto reconnect
```

Comment:
I'm actually not too sure if that's true (and if it is then its a bug IMO)

Reply:
Hi @leibale 
no one will catch if you thow here. I (painfully) experienced it :-D

I think you should generally never throw in an event handler.

If the doc makes it clear, then this is not a bug.

This is the reason why I opened this PR.

Reply:
this bug is fixed

Reply:
Good to hear ;-)

Out of curiosity, do you have a commit id? I wonder how it was fixed...

Anyway the comment was removed. PR is ready.